exercises/G-conditionals-2/exercise-2.js

```diff
@@ -7,7 +7,14 @@
 
 */
 
-function studentPassed(grade) {}
+function studentPassed(grade) {
+  if (grade < 50) {
```

Comment:
Could you replace the two if statements with an if-else to simplify the logic here?

Reply:
thanks Maten!